### DISCUSSION TITLE
ChatGPT Plus or Phind Pro

### DISCUSSION THREAD

**Discussion by altugoz** [Cyfrin/foundry-full-course-cu#1239] [General]
Hi guys, which one of these subscription do you recommend: ChatGPT Plus or Phind Pro?

**Comment by xyz899** [comment]
Hi,
Definitely Chat GPT. Phind is ChatGPT 4, with the difference that Phind is better in searching the internet and providing up to date code that is not always the case since it relies heavily on internet searches. 
with ChatGPT, You have GPT-4 With DALL·E, browsing and analysis. Can search on the web, makes AI images, brainstorm ideas and is supported with more languages I use it with C++, Rust, Solidity, Test Frameworks, Web Dev, I use it for daily problems as well even served me when I was stuck on vacation months before.
 My only tip for you is when working on specialized problems always verify the information and test it before making any big changes to a project

**Reply by altugoz** [reply]
Hi,

Thanks for the answer. So coming back to coding, any idea on github copilot vs chatgpt plus? I saw on vscode you can actually integrate chatgpt itself as an extension.
